nixos/modules/services/x11/xserver.nix

```diff
@@ -812,4 +812,8 @@ in
 
   };
 
+  imports = [
+    (mkRemovedOptionModule ["services" "xserver" "useXFS" ]
+     "Use services.xserver.fontPath instead of useXFS")
+  ];
```

Comment:
```suggestion
```

This should be moved to the already-present `imports` here:

https://github.com/NixOS/nixpkgs/blob/03f43b02db536eae4c04b8677969ddfa154ab537/nixos/modules/services/x11/xserver.nix#L148

Reply:
done